libraries/mbed/targets/hal/TARGET_WIZNET/TARGET_W7500x/serial_api.c

```diff
@@ -131,11 +158,8 @@ void serial_free(serial_t *obj)
 
     if (obj->uart == UART_1) {
     }
-
-
-    // Configure GPIOs
-//    pin_function(obj->pin_tx, STM_PIN_DATA(STM_MODE_INPUT, GPIO_NOPULL, 0));
-//    pin_function(obj->pin_rx, STM_PIN_DATA(STM_MODE_INPUT, GPIO_NOPULL, 0));
+    if (obj->uart == UART_2) {
+    }
```

Comment:
I didn't prepare deinit API of UART not yet.So I will fill later, if you don't mind.